### ISSUE TITLE
"WARN: Tower request field `tasks.script` exceeds expected size" in Tower with templated modules

### ISSUE THREAD

**Issue by pinin4fjords** [nextflow-io/nextflow#3328]
### Expected behavior and actual behavior

I would expect that a workflow incorporating templated Nextflow modules like [this one](https://github.com/nf-core/modules/tree/master/modules/nf-core/deseq2/differential) would work in Tower, as they do in Nextflow outside of Tower.  

What actually happens in Paolo's words is that "the template is expanded to the actual looong script string causing the problem at tower level". 

The jobs themselves _seem_ to run okay.

### Steps to reproduce the problem

The nf-core style workflow at https://github.com/healx/nf-core-differentialabundance/tree/healx_starting_workflow can be imported to show this behaviour. Test data as recorded in the test profile there. 

(note that the workflow-running test is passing in CI there).

### Program output 

This leads to errors at the top of the console output like ```WARN: Tower request field `tasks.script` exceeds expected size``` and failures at the end like:

```
...
-[0;35m[nf-core/differentialabundance][0;32m Pipeline completed successfully[0m-
WARN: Unexpected HTTP response.
Failed to send message to https://api.tower.nf -- received 
- status code : 400
- response msg: Oops... Unable to process request - Error ID: 5kk9S3p5y3BYMIwB64aJgW
- error cause : Oops... Unable to process request - Error ID: 5kk9S3p5y3BYMIwB64aJgW
```

### Environment 

* Nextflow version: 22.10.0 
* Java version: Groovy 3.0.13 on OpenJDK 64-Bit Server VM 17.0.5+8-LTS
* Operating system: Linux 4.14.294-220.533.amzn2.x86_64
* Bash version:

**Comment by pinin4fjords** [comment]
To be clear, the workflow runs to the end, to the extent of all the files I expect appearing in my reports. It would just be good if Tower didn't mark the run as failed.

**Comment by pditommaso** [comment]
Solved by f7fe96f7
